styles/_global.scss

```diff
@@ -0,0 +1,15 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
```

Comment:
I guess setting margin and padding to zero could be useful, what do you think? 🤔

Reply:
I think it could have been useful if we did it in the beginning. There is a lot of styling done now, perhaps in the refactor or when we deal with responsiveness

Reply:
From the beginning there is implemented a normalize.css, isn't it a solution for that?

Reply:
Now I am not 100%, but I dont think Normalize resets margins and paddings, and they can sometimes make it hard to recreate the given design. All heading elements for example or other have their margins, if you reset them, I think you gain full control of what happens with elements.